Library/Formula/github-markdown-toc.rb

```diff
@@ -0,0 +1,19 @@
+class GithubMarkdownToc < Formula
+  desc "Easy TOC creation for GitHub README.md (in go)"
+  homepage "https://github.com/ekalinin/github-markdown-toc.go"
+  url "https://github.com/ekalinin/github-markdown-toc.go/archive/0.4.1.tar.gz"
+  sha256 "fb50c97b7a19152307ba142e6a9ee6f7bdb6659e3e4caa0fe38ffb69db7cf567"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "make", "get-deps"
+    system "go build -tags netgo --ldflags '-s -extldflags \"-lm -lstdc++ -static\"' -i -o gh-md-toc"
```

Comment:
Any reason we can't use `make build` here? https://github.com/ekalinin/github-markdown-toc.go/blob/master/Makefile#L24